src/persistence/settings.cpp

```diff
@@ -466,16 +489,37 @@ void Settings::loadPersonal(QString profileName, const ToxEncrypt* passKey)
     QString filePath = dir.filePath(globalSettingsFile);
 
     // load from a profile specific friend data list if possible
+    bool defaultSettings = false;
     QString tmp = dir.filePath(profileName + ".ini");
-    if (QFile(tmp).exists()) // otherwise, filePath remains the global file
+    if (QFile(tmp).exists()) { // otherwise, filePath remains the global file
         filePath = tmp;
+        defaultSettings = true;
```

Comment:
This detection is buggy, when the file exists we can determine its version from the content.

Reply:
Thanks for finding that, updated.

Reply:
The update seems to have fixed it.